.github/workflows/linux.yml

```diff
@@ -163,6 +163,35 @@ jobs:
             -DAMReX_CUDA_ARCH=6.0
         make -j 2
 
+  # Build libamrex and all tutorials with CUDA 11.0.2 (recent supported)
+  tutorials-cuda11:
+    name: CUDA@11.0.2 GNU@9.3.0 C++17 Release [tutorials]
+    runs-on: ubuntu-20.04
+    env: {CXXFLAGS: "-fno-operator-names"}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Dependencies
+      run: .github/workflows/dependencies/dependencies_nvcc11.sh
+    - name: Build & Install
+      run: |
+        export PATH=/usr/local/nvidia/bin:/usr/local/cuda/bin:${PATH}
+        export LD_LIBRARY_PATH=/usr/local/nvidia/lib:/usr/local/nvidia/lib64:${LD_LIBRARY_PATH}
```

Comment:
I think you want `/usr/local/cuda/lib64` here too.

Reply:
Already in there (2nd added argument) :)

Reply:
Oh wait, it's not. Will add!
(Curiously, the current syntax works for WarpX.)

Reply:
It's not always necessary since nvcc statically links the CUDA runtime library libcudart.so by default -- it is more relevant when you're using things like cuBLAS. All of the other driver libraries like libcuda.so and libnvidia-ml.so go in system standard places like /usr/lib64.

Reply:
Thanks for the details, will modernize WarpX' CI then as well: https://github.com/ECP-WarpX/WarpX/pull/1581